KubeArmor/monitor/systemMonitor.go

```diff
@@ -629,10 +629,11 @@ func (mon *SystemMonitor) TraceSyscall() {
 
 			// Best effort replay
 			go func() {
-				time.Sleep(1 * time.Second)
```

Comment:
Lets keep this sleep as it is needed for container logic.

Reply:
OK！